### PR TITLE
Adding AppInsights Instrumentation Key env Variable for Diagnostics MS

### PR DESCRIPTION
**Pull request by isaac-dasan**
# Description and Motivation 
Adding AppInsights Instrumentation Key env Variable for Diagnostics MS
...

# Change type 

- [ ] Bug fix
- [ ] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

---
This change is [Reviewable](https://reviewable.io/reviews/azure/pcs-cli/468)

### CODE REVIEW

File: src/deploymentmanager.ts [hunk 1]
```diff
@@ -420,11 +420,11 @@ export class DeploymentManager implements IDeploymentManager {
         } else if (this._template.parameters.deploymentId) {
             this._parameters.deploymentId = { value: answers.deploymentId };
         }
-        if (answers.diagnosticsEndpointUrl) {
-            if (this._parameters.diagnosticsEndpointUrl) {
-                this._parameters.diagnosticsEndpointUrl.value = answers.diagnosticsEndpointUrl;
-            } else if (this._template.parameters.diagnosticsEndpointUrl) {
-                this._parameters.diagnosticsEndpointUrl =  { value: answers.diagnosticsEndpointUrl };
+        if (answers.appInsightsInstrumentationKey) {
+            if (this._parameters.appInsightsInstrumentationKey) {
```
Comment (ppathan) [inline src/deploymentmanager.ts]: Can you add this value to parameters.json?

File: solutions/remotemonitoring/single-vm/docker-compose.java.yml [hunk 2]
```diff
@@ -198,7 +198,7 @@ services:
       - PCS_TELEMETRY_STORAGE_TYPE
 
   diagnostics:
-    image: azureiotpcs/pcs-diagnostics-dotnet:${PCS_DOCKER_TAG}
+    image: azureiotpcs/pcs-diagnostics-dotnet:Diagnostics-2.0.0
```
Comment (ppathan) [inline solutions/remotemonitoring/single-vm/docker-compose.java.yml]: Is this going to be new Diagnostics service version?

Comment (isaac-dasan) [reply solutions/remotemonitoring/single-vm/docker-compose.java.yml]: yes

File: src/deploymentmanager.ts (hunk 1 above)
Comment (isaac-dasan) [reply src/deploymentmanager.ts]: added standard-parameters.json

**Pull request merged**